src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToStoreAProjectWithDependencies.cs

```diff
@@ -283,15 +283,16 @@ public void It_creates_profiling_symbols()
             {
                 // profiling symbols are not supported on OSX
                 symbolsFolder.Should().NotExist();
-                return;
             }
+            else
+            {
+                var newtonsoftSymbolsFolder = symbolsFolder.Sub("newtonsoft.json").Sub("9.0.1").Sub("lib").Sub("netstandard1.0");
+                newtonsoftSymbolsFolder.Should().Exist();
 
-            var newtonsoftSymbolsFolder = symbolsFolder.Sub("newtonsoft.json").Sub("9.0.1").Sub("lib").Sub("netstandard1.0");
-            newtonsoftSymbolsFolder.Should().Exist();
-
-            var newtonsoftSymbolsFiles = newtonsoftSymbolsFolder.GetFiles().ToArray();
-            newtonsoftSymbolsFiles.Length.Should().Be(1);
-            newtonsoftSymbolsFiles[0].Name.Should().StartWith("Newtonsoft.Json").And.EndWith(symbolFileExtension);
+                var newtonsoftSymbolsFiles = newtonsoftSymbolsFolder.GetFiles().ToArray();
+                newtonsoftSymbolsFiles.Length.Should().Be(1);
+                newtonsoftSymbolsFiles[0].Name.Should().StartWith("Newtonsoft.Json").And.EndWith(symbolFileExtension);
+            }
```

Comment:
Why this change?

Reply:
There is a blind "return;" 
The test should execute block A or block B based on the conditional "RuntimeInformation.IsOSPlatform(OSPlatform.OSX)"- essentially the return is acting as the 'else'; therefore, I added it.